src/pages/Team/components/List/List.tsx

```diff
@@ -1,13 +1,67 @@
-import React from 'react';
+import { FormControl, MenuItem, Select } from '@material-ui/core';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import React, { useEffect, useRef, useState } from 'react';
 
 import { IncomeDb } from 'models/income.models';
+
 import './style.css';
 
-export const List = ({ rows }: {rows: IncomeDb[]}): JSX.Element => {
+export const List = ({ navHeight, scrollPosition, rows }: {navHeight: number | null, scrollPosition: number, rows: IncomeDb[]}): JSX.Element => {
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(25);
+  const [displayRows, setDisplayRows] = useState<IncomeDb[]>(rows);
+  const [barFixed, serBarFixed] = useState<boolean>(false);
+  const [heightFirstLi, setHeightFirstLi] = useState<number>(0);
+
+  const handleRowsPerPage = (value: number | string) => { 
+    setRowsPerPage(Number(value));
+
+    if ((page * rowsPerPage) > displayRows.length) {
+      setPage(0);
+    }
+
+  };
+
+  useEffect(() => {
+    if (navHeight && navHeight < scrollPosition) {
+      if (firstLi?.current?.clientHeight) {
+        setHeightFirstLi(firstLi?.current?.clientHeight);
+      }
+      serBarFixed(true);
+    } else {
+      setHeightFirstLi(0);
+      serBarFixed(false);
+    };
+  },[scrollPosition]);
+
+  useEffect(() => {
+    const min = page * rowsPerPage;
+    const max = min + rowsPerPage;
+    const copyRows = [...rows].slice(min,max);
+    setDisplayRows(copyRows);
+  },[rowsPerPage, page, rows]);
+  
+  useEffect(() => {
+    window.scrollTo(0,0);
+  },[page]);
+
+  const handleChangePage = (action: string) => {
+    
+    if (action === 'next' && rows.length > page * rowsPerPage + rowsPerPage) {
```

Comment:
Here we should use enum (it can be declared locally, if we do not use its in other place)

Reply:
You're right. Done.